rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Please make sure to assign the rentals to the book and person instances for the record to be saved in the Rental model class when it's invoked by the other two. 🙏 

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self

Reply:
Yeah sure. 😄

Reply:
issue resolved! ✔️